### DISCUSSION TITLE
Android App stop at PushNotifications.register()

### DISCUSSION THREAD

**Discussion by CedrickTAGNE** [ionic-team/capacitor#6990] [Q&A] [answered]
Hi,
Please I'm facing an issue with capacitor PushNotifications plugin.
I added it it in my app, but when the execution reach the PushNotifications.register() step on Android, the app is closed without any error.

I'm using Angular Ionic 5.
Please does anyone has an idea on how to fix that? It's difficult to check what's the issue since there is no error returned.

Here is my code
`
PushNotifications.requestPermissions().then((result) => {
      if (result.receive === "granted") {
        PushNotifications.register().then((v) =>
          console.log("PushNotification granted and registered")
        );
      } else {
        // Show some error
      }
    });
`
Thanks

**Comment by LennonReid** [comment]
Try to use https://ionicframework.com/docs/v5/angular/your-first-app/live-reload to debug.

**Comment by jcesarmobile** [comment, marked as answer]
no errors where? if the app is closing you should see an error in logcat

push crash on register can happen if you didn't add the `google-services.json` file or if it doesn't match your app identifier

**Reply by CedrickTAGNE** [reply]
I'm not running from Android studio, I'm using the chrome device inspect to debug. And as it's closing right after the Register. checking the inspector console, there is nothing printed there.
Also I already have the google-service.json in android folder. btw I think it's related to FCM, but I still at the PushNotification.register, I'm not yet calling any FCM function.

**Reply by jcesarmobile** [reply]
Native errors won’t appear on chrome inspector, you have to check logcat, even if not using android studio to run.

**Reply by CedrickTAGNE** [reply]
Thanks,
I check in the log cat I'm having the errorFirebaseApp is not initialized. I was putting  `google-services.json` in the wrong directory, it's under android/app I was putting it under android/

Thanks a lot for your answers and your time
